sensors_script/device_availability_monitor.py

```diff
@@ -0,0 +1,78 @@
+"""
+Module that monitor package installation and control LEDs accordingly  
+"""
+import RPi.GPIO as GPIO
+import time
+import subprocess
+
+GPIO.setmode(GPIO.BCM)
+
+GREEN_LED = 10
+YELLOW_LED = 9
+RED_LED = 11
+
+DPKG_LOG_PATH = '/var/log/dpkg.log'
+UPFLUX_LOG_PATH = '/var/log/upflux.log'
+
+GPIO.setup(GREEN_LED, GPIO.OUT)
+GPIO.setup(YELLOW_LED, GPIO.OUT)
+GPIO.setup(RED_LED, GPIO.OUT)
+
+def set_green_default():
+    GPIO.output(GREEN_LED, GPIO.HIGH)
+    GPIO.output(YELLOW_LED, GPIO.LOW)
+    GPIO.output(RED_LED, GPIO.LOW)
+
+def is_package_installing():
```

Comment:
Has this been tested? using a package??

Reply:
Yes it has

Reply:
Updated description, i think you would be able to see it better in person since you can see the hardware reacting